website/docs/r/site.markdown

```diff
@@ -7,118 +7,196 @@ description: |-
-#### excluded_subnets
-Network subnets to exclude when Default Gateway is enabled. The traffic for these subnets will not go through the Gateway in this Site. Deprecated as of 6.0. Use action type &#39;exclude&#39; in Entitlements instead.
+
 ### vpn
 VPN configuration for this Site.
-
-* `state_sharing`:  (Optional)  default value `false` Configuration for keeping track of states.
-* `snat`:  (Optional)  default value `false` Source NAT.
+* `state_sharing`:  (Required)  default value `false` Configuration for keeping track of states.
+* `snat`:  (Required)  default value `false` Source NAT.
 * `tls`:  (Optional) VPN over TLS protocol configuration.
 * `dtls`:  (Optional) VPN over DTLS protocol configuration.
 * `route_via`:  (Optional) Override routing for tunnel traffic.
-* `url_access_enabled`:  (Optional) Whether to enable URL Access feature or not.
-* `url_access_p12s`:  (Optional) P12 files for proxying traffic for URL Access feature.
 * `ip_access_log_interval_seconds`:  (Optional)  default value `120` Frequency configuration for generating IP Access audit logs for a connection.
+
 #### tls
 VPN over TLS protocol configuration.
-* `enabled`: (Optional)
+* `enabled`: (Required) Whether to enable tls
+
 #### dtls
 VPN over DTLS protocol configuration.
-* `enabled`: (Optional)
+* `enabled`: (Required) Whether to enable dtls
+
 #### route_via
 Override routing for tunnel traffic.
 * `ipv4`: (Optional) IPv4 address for routing tunnel traffic. Example: 10.0.0.2.
 * `ipv6`: (Optional) IPv6 address for routing tunnel traffic. Example: 2001:db8:0:0:0:ff00:42:8329.
-#### url_access_p12s
-P12 files for proxying traffic for URL Access feature.
-* `id`: (Optional) Identifier to track the object on update since all the other fields are write-only. A random one will be assigned if left empty.
-* `content`: (Optional) Contents of the P12 file in Base64 format.
-* `password`: (Optional) Password for the P12 file.
-* `subject_name`: (Optional) Subject name of the certificate in the file.
-* `verify_upstream`: (Optional) Gateway will verify upstream certificate of the endpoints.
+
 ### name_resolution
 Settings for asset name resolution.
-
 * `use_hosts_file`:  (Optional)  default value `false` Name resolution to use Appliance's /etc/hosts file.
 * `dns_resolvers`:  (Optional) Resolver to resolve hostnames using DNS servers.
 * `aws_resolvers`:  (Optional) Resolvers to resolve Amazon machines by querying Amazon Web Services.
 * `azure_resolvers`:  (Optional) Resolvers to resolve Azure machines by querying Azure App Service.
 * `esx_resolvers`:  (Optional) Resolvers to resolve VMware vSphere machines by querying the vCenter.
 * `gcp_resolvers`:  (Optional) Resolvers to resolve GCP machine by querying Google web services.
 * `dns_forwarding`:  (Optional) Enable DNS Forwarding feature.
+
+```hcl
+
+```
```

Comment:
Forgotten code block?

Reply:
Tag is already listed in the top-level argument reference